js/apps/events-page/constants.js

```diff
@@ -0,0 +1,19 @@
+((app) => {
+	'use strict';
+
+	var constants = {
+		// baseUrl: 'https://testservices.bcpl.info',
+		baseUrl: 'http://ba224964:3100',
+		serviceUrls: {
+			events: '/api/evanced/signup/events'
+		},
+		templateUrls: {
+			event: '/dist/js/apps/events-page/templates/event.html',
```

Comment:
rename "rerserved-ish" term per https://www.w3schools.com/js/js_reserved.asp

Reply:
Which term?

Reply:
```event```

Reply:
Renamed them all to directiveNameTemplate.